independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/Dockerfile-layout.include.qute

```diff
@@ -20,16 +20,71 @@
 #
 # docker run -i --rm -p 8080:8080 quarkus/{project.artifact-id}-{type}
 #
+# This image uses the `run-java.sh` script to run the application.
```

Comment:
These docs are fine by now but really should consider putting this in a guide somewhere and link to it instead.